src/test/java/jdbchomework/controller/CompanyControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package jdbchomework.controller;
+
+import jdbchomework.service.CompanyService;
+import org.junit.Before;
+import org.junit.Test;
+
+import static jdbchomework.utils.TestUtils.ALL_COMPANIES;
+import static jdbchomework.utils.TestUtils.ID;
+import static jdbchomework.utils.TestUtils.NAME;
+import static jdbchomework.utils.TestUtils.COMPANY_RESULT;
+
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+
+public class CompanyControllerTest {
+    private CompanyService companyService = mock(CompanyService.class);
+    private CompanyController companyController;
+
+
+    @Before
+    public void setUp() {
+        companyController = new CompanyController(companyService);
+    }
+
+    @Test
+    public void shouldGetAllCompanies() {
+        when(companyService.getAllCompanies()).thenReturn(ALL_COMPANIES);
+        assertEquals(ALL_COMPANIES, companyController.getAllCompanies());
+        verify(companyService, times(1)).getAllCompanies();
```

Comment:
Considering that you check method result, this line is actually not needed - it checks the same